### DISCUSSION TITLE
How to disable the preview tab?

### DISCUSSION THREAD

**Discussion by ackmanx** [zed-industries/zed#7392] [Feature Requests]
One of the first things I do in VS Code is require a double click to open a file. It's very disorienting for me to see tabs open all the time just by clicking on a file in a tree. One example is copy and paste, which ends up opening a tab. Intellij introduced this feature to copy VS Code, but also lets the user disable it. 

Is there a setting to disable this similar behavior on Zed? I searched the default settings with terms "tab" and "preview" but did not see anything.

**Comment by Moshyfawn** [comment]
There is issue #4922 tracking this feature request. There are no alternatives in Zed at the moment.

**Comment by d1y** [comment]
https://github.com/zed-industries/zed/pull/6782

**Comment by JosephTLyons** [comment]
Hey @ackmanx - the snippet you are looking for is:

```json
"preview_tabs": {
  "enabled": false
}
```
